src/sentry/static/sentry/app/components/quickTrace/index.tsx

```diff
@@ -151,6 +158,47 @@ export default function QuickTrace({
     />
   );
 
+  if (traceLength === 1) {
+    nodes.push(
+      <Projects
+        key="missing-services"
+        orgId={organization.slug}
+        slugs={[current.project_slug]}
+      >
+        {({projects}) => {
+          const project = projects.find(p => p.slug === current.project_slug);
+          if (project) {
```

Comment:
This way you won't need the optional chaining or type casting below.

```suggestion
          if (project?.platform) {
```